starter-core/src/main/java/io/micronaut/starter/feature/build/gradle/GradleEnterpriseConfiguration.java

```diff
@@ -34,7 +34,7 @@ default String getTermsOfServiceUrl() {
     }
 
     @Nullable
-    default String getTermsOfServiceAgree() {
-        return "yes";
+    default Boolean getTermsOfServiceAgree() {
```

Comment:
this is a breaking change.

Reply:
I have created a PR on top of yours deprecating this method instead of removing it https://github.com/micronaut-projects/micronaut-starter/pull/1595/commits/6f356eb6b5915b46c9ab372060558bf8a634b219

are you ok wit this change @alextu ?

Reply:
Sure, no problem